Samples/iOS-Swift/iOS-Swift/Profiling/ProfilingViewController.swift

```diff
@@ -182,4 +145,45 @@ class ProfilingViewController: UIViewController, UITextFieldDelegate {
             profilingUITestDataMarshalingStatus.text = "❌"
         }
     }
+
+    func optionsConfiguration() {
+        guard let options = SentrySDK.currentHub().getClient()?.options else { return }
+
+        if let sampleRate = options.profilesSampleRate {
+            sampleRateField.text = String(format: "%.2f", sampleRate.floatValue)
+            sampleRateField.isEnabled = true
+            profilesSampleRateSwitch.isOn = true
+        } else {
+            sampleRateField.isEnabled = false
+            sampleRateField.text = "nil"
+            profilesSampleRateSwitch.isOn = false
+        }
+
+        if let sampleRate = options.tracesSampleRate {
+            tracesSampleRateField.text = String(format: "%.2f", sampleRate.floatValue)
+            tracesSampleRateField.isEnabled = true
+            tracesSampleRateSwitch.isOn = true
+        } else {
+            tracesSampleRateField.text = "nil"
+            tracesSampleRateField.isEnabled = false
+            tracesSampleRateSwitch.isOn = false
+        }
+
+        if let v2Options = options.profiling {
```

Comment:
`l`: This reads a bit better, IMO.
```suggestion
        if let optionsV2 = options.profiling {
```